### DISCUSSION TITLE
https://thanhle.blog/vi/blog/my-2021

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#13] [Ideas]
# https://thanhle.blog/vi/blog/my-2021

https://thanhle.blog/vi/blog/my-2021

**Comment by darkpiv** [comment]
Ngon em, kiếm thêm ít ít tiền dư sống rồi về trồng cà phê code vì đam mê hoy.

**Comment by richard-xtek** [comment]
Hay m ơi. Chúc m năm 2022 gặt hái được nhiều thành công nhé. :D kkk
